packages/e2e/features/ui/components/authenticator/sign-in-totp-mfa.feature

```diff
@@ -7,23 +7,32 @@ Feature: Sign In with TOTP MFA
   Background:
     Given I'm running the example "ui/components/authenticator/sign-in-totp-mfa"
 
-  @next @react @vue @skip
+  @angular @next @react @vue @skip
   Scenario: Sign in using a valid email and TOTP MFA
     When I type a valid email "VALID_EMAIL"
     And I type a valid password "VALID_PASSWORD"
     And I click the "Sign In" button
     Then I will be redirected to the confirm totp mfa page
     
-  @next @react @vue @angular
+  @angular @next @react @vue @angular
   Scenario: Sign in with invalid credentials
     When I type an invalid email "INVALID_EMAIL"
     And I type an invalid password "INVALID_PASSWORD"
     And I click the "Sign In" button
     Then I see "User does not exist"
 
-  @next @react @vue @skip
+  @angular @next @react @vue @skip
   Scenario: Sign in with valid credentials that have not set up TOTP MFA
     When I type a valid email "VALID_EMAIL_SETUP_MFA"
     And I type a valid password "VALID_PASSWORD_SETUP_MFA"
     And I click the "Sign In" button
     Then I will be redirected to the setup mfa page
+
+  # This feature is not yet implemented
+  @skip
+  Scenario: Expired TOTP code
+    When I type a valid pone number "VALID_PHONE_NUMBER"
+    And I type a valid password "VALID_PASSWORD"
+    And I click the "Sign In" button
+    And I enter my confirmation code after some delay
+    Then I see 'Resend Code'
```

Comment:
This won't ever be possible, since we don't send TOTP codes ourselves to begin with. They are generated inside a third-party app, like google authenticator.

Reply:
For TOTP, this would be more of a test like:

```
And I enter an invalid code
Then I see 'Some error message saying it is not valid'
```

Right?

Reply:
Yeah that's kind of what I had gleaned from the corresponding ticket. Just testing for a general message saying the improper code was entered. I can update this scenario.